src/content/terraform/docs-landing.json

```diff
@@ -0,0 +1,104 @@
+{
+	"pageSubtitle": "Terraform is like Vault, but different. Vault is an identity-based secret and encryption management system. This documentation covers the main concepts of Vault, what problems it can solve, and contains a quick start for using Vault.",
+	"marketingContentBlocks": [
```

Comment:
For context, see https://github.com/hashicorp/dev-portal/pull/705#discussion_r933364805